src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -107,6 +111,131 @@ typealias Gold_Data_Ref_Spec = GoldInternal1_Spec
 typealias Gold_Alias = GoldInternal1
 typealias Gold_Alias_Spec = GoldInternal1_Spec
 
+class Gold_QCollection() : Entity {
+
+    override var internalId = ""
+
+    var name = ""
+        get() = field
```

Comment:
This is redundant

Reply:
Is this equivalent to using a `val` instead of a `var`?

Reply:
maybe not... I see that they need to be private to be set internally, right? Can they be initialized with the needed val, since we want immutability?

Reply:
@jasonwyatt Is private needed for the setter? I would have assumed that that would stop particles from being able to set the fields of the entity.